specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/examples/VirtualNetworkGatewayConnectionUpdateTags.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters" : {
+    "api-version": "2018-07-01",
+    "subscriptionId" : "subid",
+    "resourceGroupName" : "rg1",
+    "virtualNetworkGatewayConnectionName": "test",
+    "parameters": {
+      "tags": {
+        "tag1": "value1",
+        "tag2": "value2"
+      }
+    }
+  },
+  "responses" : {
+    "200" : {
+      "body" : {
+        "name": "test",
+        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/connections/test",
+        "type": "Microsoft.Network/connections",
+        "location": "westus",
+        "tags": {
+          "tag1": "value1",
+          "tag2": "value2"
+        },
+        "properties": {
+          "provisioningState": "Succeeded",
+          "resourceGuid": "00000000-0000-0000-0000-000000000000",
+          "virtualNetworkGateway1": {
+            "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/virtualNetworkGateways/vpngw"
```

Comment:
According to https://github.com/Azure/azure-rest-api-specs/pull/3635/files#diff-795d13db3a317a9440687541d8d00549R2545, `properties` property is required.